packages/generators/init-template/default/postcss.config.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+    // Add you postcss configuration here
+    // Learn more about it at https://github.com/webpack-contrib/postcss-loader#config-files
+    plugins: [['postcss-preset-env']],
```

Comment:
I think better put `autoprefixer` here, it is more popular

Reply:
I found one more thing while exploring, `autoprefixer` is already included in `postcss-preset-env` (https://github.com/csstools/postcss-preset-env#autoprefixer).

Reply:
`postcss-preset-env` is very poorly updated and contains rather controversial solutions, if someone wants to add it, he can do it very easily, most developers do not use it due a lot of bugs and problems